chart/k8gb/templates/operator.yaml

```diff
@@ -121,3 +121,5 @@ spec:
               value: {{ quote .Values.k8gb.log.level }}
             - name: NO_COLOR
               value: "true"
+            - name: NO_SPLIT_BRAIN
```

Comment:
Split-brain is the situation that we might want to avoid.  In that case, it makes sense to name the feature flag as `SPLIT_BRAIN_PREVENTION`. It also makes sense to disable the `SPLIT_BRAIN_PREVENTION` flag by default.